repo/meta/schema/package-schema.json

```diff
@@ -68,6 +72,7 @@
     }
   },
```

Comment:
Yes. We already removed it. It is part of the version-2 branch. This PR is against version-2. I am fixing the documentation and adding a `packagingVersion` property.